src/components/linear-progress/linear-progress.tsx

```diff
@@ -57,9 +57,10 @@ export class LinearProgress {
 
     public render() {
         /**
-         * Avoid rendering the component when the prop 'value' is explicitly set to null.
+         * Avoid rendering the component when the prop 'value' explicitly set to null,
+         * undefined or object.
```

Comment:
This is a good example of why comments in code is almost never a good idea :smile: 
We made a change in the code, which forced us to also update the comment. However, the comment is not actually explaining what the code is doing and is confusing rather than helping. If we have a comment here it should probably say something like "Avoid rendering unless the value is a number". This is obvious from the code though, so there is no need for a comment at all. 
Comments in code is only warranted to answer _why_ it is doing something. Not _what_ it is doing, since that should already be obvious by reading the code.